package.json

```diff
@@ -90,7 +90,7 @@
     "opn": "^5.3.0",
     "postcss-js": "^1.0.1",
     "postcss-loader": "^2.1.6",
-    "prettier": "~1.9.0",
+    "prettier": "1.14.2",
```

Comment:
Why pin this? Do we not trust their semantic versioning?

Is it bad if minor releases of Prettier include rule changes that break linting? Isn't that what `sku format` is for?

Reply:
Changes to prettier version break the `sku lint` command, breaking consumer builds.

We don't have to lock it but it might negatively hurt the reliability of sku as a product if consumers find that one day their builds pass and one day it doesn't.

Will discuss offline and see what comes of it.